api/pyproject.toml

```diff
@@ -108,7 +108,8 @@ authlib = "1.3.1"
 azure-identity = "1.16.1"
 azure-storage-blob = "12.13.0"
 beautifulsoup4 = "4.12.2"
-boto3 = "1.34.136"
+boto3 = "1.34.148"
+botocore = "^1.34.148"
```

Comment:
Avoid transparent dependencies since `botocore` is already required by `boto3`.

Reply:
Done